### ISSUE TITLE
Handle special characters

### ISSUE THREAD

**Issue by misterbenj34** [marian-m12l/studio#29]
Special characters har not handled by the interface.
```
à are shown as Ã
é are shown as Ã©
ê are shown as Ãª
- are shown as â€“
```

**Comment by marian-m12l** [comment]
Where exactly are you seeing this? As the saying goes, "works on my machine" ;-) What is your OS/browser?

**Comment by misterbenj34** [comment]
![bug_spe_car](https://user-images.githubusercontent.com/24669993/71627507-1ce46980-2bf3-11ea-9495-a3ce82077a6a.PNG)
Here what I see

I'm on Windows 10, Chrome Version 79.0.3945.88 (64 bits) (in French)

**Comment by dasebasto** [comment]
I confirm there is a character encoding issue.

Accents are displayed correctly in the editor (even after reloading the pack) :
![image](https://user-images.githubusercontent.com/4549348/78078669-eca39700-73aa-11ea-908a-51c965c5acb9.png)

But not in the library:
![image](https://user-images.githubusercontent.com/4549348/78078641-de557b00-73aa-11ea-8c6c-388050b81266.png)

**Comment by marian-m12l** [comment]
Thanks for the additional info. Looks like it affects both official and unofficial metadata, but just to be sure can you check the encoding of `.studio/db/official.json` and `.studio/db/unofficial.json`?

**Comment by dasebasto** [comment]
`official.json` has ISO-8859-1 encoding (I don't have special characters in my official library, so I cannot confirm I have the bug).

`unofficial.json` has UTF-8 encoding

STUdio displays names from `unofficial.json` as if they were encoded in ISO-8859-1.

**Comment by marian-m12l** [comment]
Character encoding should (hopefully) be fixed on master. Can you give it a try?

**Comment by marian-m12l** [comment]
fixed with release 0.1.13
